sceptre/scipool/templates/bmgfki/lambda-budgets/budget-rules.yaml

```diff
@@ -0,0 +1,8 @@
+# budget rules per user by synapse team id
+teams:
+  '3432633': # BMGFKI_ServiceCatalogUsers
+    amount: '100'
+    period: ANNUALLY
+    unit: USD
+    community_manager_emails:
+      - 'sc-support@sagebase.org'
```

Comment:
`sc-support@sagebase.org` is a Google group.  Anna Greenwood is a member but Brian Bot is not.  What do you think of having a Group, `bmgf-ki-sc-support@sagebase.org` that would be used here, with Brian added to the group?

Reply:
group created, https://groups.google.com/a/sagebase.org/g/bmgfki-sc-support